test_queue_job/models/test_models.py

```diff
@@ -44,6 +45,10 @@ def testing_method(self, *args, **kwargs):
             return self.env.context
         return args, kwargs
 
+    def test_long_running_method(self):
```

Comment:
what is this test for?

Reply:
Terminating a running job can only be tested if you have something that lasts long enough for a job to be queued and then started. Hence this "sleep trigger on the DB to have a long running DB transaction.
The reason I added this was for practical testing. An automated test would be nice of course, but would require a threaded test so the job can be started and then terminated by a different process. I did not have the time to spend on this to write this kind of test.

Reply:
Probably you can mock it instead. In this case I don't see the benefit if not let tests take longer than they should.

Reply:
@simahawk the tests do not take longer since this method is not tested, it is there for manual configuration. It is a sample to ensure the functionality can be tested.

Reply:
ah I see... method name cheated me :sweat_smile: 
Ok, then I see no reason to keep it. If you really want to keep it please add an extended message explaining the purpose. Also, rename it to `testing_*`.

Reply:
Thanks for the improvements, I really appreciate it!
I do prefer to keep this method so I updated the method name and added the docstring.